pkg/gitversion/gitversion.go

```diff
@@ -130,12 +132,23 @@ func GetLanguageVersionsWithOptions(opts LanguageVersionsOptions) (*LanguageVers
 		preVersion = fmt.Sprintf("%s%sdirty", preVersion, separator)
 	}
 
+	buildVersion := []byte{}
+	if len(genericVersion.Build) > 0 {
```

Comment:
What happens if the `genericVersion` is a prerelease? I see the tests pass, but what is keeping this from adding two build tags to a prerelease? we do add a build tag extension to the `preVersion` [here](https://github.com/pulumi/pulumictl/pull/65/files#diff-e4c3eb5b535260361d246e905dc80ea851aac4c369cc0d08f41df0c9c897cf85R106).

Reply:
@guineveresaenger This is just weird behaviour in this code and I dare not add a test for that or touch it, because pulumictl shouldn't be adding the commit hash to prerelease tags.